.github/workflows/build-container-images.yml

```diff
@@ -0,0 +1,44 @@
+name: Build indicator container images and upload to registry
+
+on:
+  push:
+    branches: [ main, prod ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        packages: [""]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: cmu-delphi-deploy-machine
+          password: ${{ secrets.CMU_DELPHI_DEPLOY_MACHINE_PAT }}
+
+      - name: Build, tag, and push image to Github
+        env:
```

Comment:
an alternative is changing the working directory directly before the run. Then you don't need docker file specification and the changing directory afterwards